apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.component.ts

```diff
@@ -240,6 +240,17 @@ export class AssetProfileDialog implements OnDestroy, OnInit {
       });
   }
 
+  public onTestScraper(symbol: string) {
+    const today = new Date();
+    this.adminService
+      .gatherSymbol({ dataSource: 'MANUAL', date: today, symbol })
+      .pipe(takeUntil(this.unsubscribeSubject))
+      .subscribe((response) => {
```

Comment:
`.subscribe(({ marketPrice }) => {`

Reply:
After providing the workaround https://github.com/ghostfolio/ghostfolio/pull/2546#discussion_r1375388256, type error is showing up, though the data is being returned.
Error:
`Property 'marketPrice' does not exist on type 'void | { createdAt: Date; dataSource: DataSource; date: Date; id: string; marketPrice: number; state: MarketDataState; symbol: string; }'.`

Data returned:
```
createdAt: null
dataSource: "MANUAL"
date: "2023-10-29T08:00:58.583Z"
id: null
marketPrice: 113.7639
state: "INTRADAY"
symbol: "trial2"
[[Prototype]]: Object
```